tests/Serializer/AbstractSerializerTest.php

```diff
@@ -346,7 +346,7 @@ public function testSerializeValueResource(bool $serializeAllObjects): void
 
         $this->assertNotFalse($filename, 'Temp file creation failed');
 
-        $resource = fopen($filename, 'wb');
```

Comment:
Please check that your local dependencies are updated, with the latest version of PHPCS-Fixer this change should not happen (it was made by @HazAT when he committed the file, then with my latest commit it got reverted back and now you're again reverting it)